### PR TITLE
December 13 2023 - creating Alex Earl spotlight

### PR DESCRIPTION
**Pull request by kmartens27**
This pull request is for the creation of @slide contributor spotlight page.

The date included in the title is the planned publication date of the spotlight page. This will be 2 weeks from the publication of the previous spotlight (going forward). The date of December 4 may change, depending on when the first spotlight is published. If that needs to be updated, it will be adjusted accordingly.

@slide is there another image that you would be okay with using? Currently, the profile picture on GitHub only includes your eyes/forehead area, and ideally we would have clear images to use. However, if you are not comfortable with that or have any reservations about this, we can proceed with what we have! Additionally, please provide any feedback or suggestions you might have and I'll be sure to update things as needed.

### CODE REVIEW

**Comment by slide** [comment]
How about this one?

**Comment by kmartens27** [comment]
That's fantastic, thanks very much @slide !

I've downloaded the image and added it accordingly

**Comment by lemeurherve** [comment]
Is it normal that the image is duplicated?
Ideally, this image should be resized and optimized with compressor.io or similar.

**Comment by kmartens27** [comment]
@lemeurherve I definitely need to run it through compressor.io & resize it a bit more.

I put it in two places because there are a couple of different areas where avatars can be stored, so I was just trying to cover everything.

**Comment by kmartens27** [comment]
@lemeurherve I was able to resize and compress the image down to ~200kb

**Review by MargoBGH — APPROVED** [review]
LGTM!

**Review by krisstern — APPROVED** [review]
LGTM

File: src/images/avatar/alex-earl.jpg
Comment (lemeurherve) [inline src/images/avatar/alex-earl.jpg:1]: Cf https://github.com/jenkins-infra/jenkins-contributor-spotlight/pull/9 this image can be removed.

Comment (kmartens27) [reply src/images/avatar/alex-earl.jpg:1]: Thanks @lemeurherve

**Comment by lemeurherve** [comment]
Preview: https://deploy-preview-7--contributor-spotlight.netlify.app/pages/contributors/alex-earl/

**Review by krisstern — APPROVED** [review]

**Review by lemeurherve — CHANGES_REQUESTED** [review]
Requesting changes to prevent merging this PR until a week or two.

cc @kmartens27

**Review by alyssat — APPROVED** [review]
LGTM!

File: src/pages/contributors/alex-earl.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+:page-twitter: 
+:page-github: slide
+:page-threads: slide_o_mix
+:page-email: 
+:page-image: avatar/alex-earl.jpg
+:page-pronouns: He/Him/his
+:page-location: Chandler, Arizona USA
+:page-firstcommit: 2008
+:page-datepublished: 2023-12-11
+:page-featured: false
```
Comment (krisstern) [inline src/pages/contributors/alex-earl.adoc:13]:
```suggestion
:page-featured: true
```

Comment (krisstern) [reply src/pages/contributors/alex-earl.adoc:13]: Then we will have to set Alex Brandes's to `:page-featured: false`

**Review by krisstern — CHANGES_REQUESTED** [review]

File: src/pages/contributors/alex-earl.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
+= Alex Earl
+:page-name: Alex Earl
+:page-linkedin: 
+:page-twitter: 
+:page-github: slide
+:page-threads: slide_o_mix
+:page-email: 
+:page-image: avatar/alex-earl.jpg
+:page-pronouns: He/Him/his
```
Comment (krisstern) [inline src/pages/contributors/alex-earl.adoc:9]:
```suggestion
:page-pronouns: He/Him/His
```

**Comment by krisstern** [comment]
Also @kmartens27 Should the publication date on which this spotlight goes out to be 2023-12-13, which is a Wednesday, or 2023-12-11, which is a Monday. I am asking because I see the others are published on a Wednesday and this one seems not to follow the prescribed pattern. Could you confirm?

**Comment by kmartens27** [comment]
@krisstern thanks for pointing that out. I had originally set it as though it would be published on a Monday. However, since we have the updated dates, I have updated this accordingly so that it matches the others.

**Comment by krisstern** [comment]
@kmartens27 Is it okay for me to push a change to this PR? I see two contributors simultaneously featured but I thought we are supposed to spotlight only one at a time and would like to correct this.

**Comment by kmartens27** [comment]
@krisstern yes of course! Always feel free to make any changes you see fit. Since this has not been merged yet though, would the featured status matter? I was envisioning leaving this the way it is, and then changing the previous contributor to `false` prior to publishing this one (the next planned one). Unless there is something I'm not considering, would that be possible/appropriate?

**Comment by krisstern** [comment]
> @krisstern yes of course! Always feel free to make any changes you see fit. Since this has not been merged yet though, would the featured status matter? I was envisioning leaving this the way it is, and then changing the previous contributor to `false` prior to publishing this one (the next planned one). Unless there is something I'm not considering, would that be possible/appropriate?

@kmartens27 I think it would be better to disable the spotlight for the previously featured contributor whilst we spotlight a new contributor, that way we keep it one-at-a-time. I have made the changes accordingly. Please see preview to see if this looks as expected. I will do the same for the next spotlight as well later once this one rolls out.

**Review by krisstern — APPROVED** [review]

**Comment by kmartens27** [comment]
Hi @krisstern that makes sense! I think we're on the same page and the change is correct. Thanks for setting that up accordingly!

**Comment by lemeurherve** [comment]
@kmartens27 @krisstern should I approve this PR to unblock it or should it stay as it is until next week?

**Review by lemeurherve — APPROVED** [review]

**Comment by krisstern** [comment]
The preview is looking great! @kmartens27 if we are publishing this today as scheduled we are all good to go!

**Comment by kmartens27** [comment]
thanks very much @krisstern, I'm merging this now so that we can get this live & send out the announcement tweet

**Pull request merged**